src/KafkaJanitor.IntegrationTests/TopicCreatedScenario.cs

```diff
@@ -0,0 +1,58 @@
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using KafkaJanitor.IntegrationTests.Utils;
+using KafkaJanitor.WebApp.Infrastructure.Messaging;
+using KafkaJanitor.WebApp.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.EnvironmentVariables;
+using Xunit;
+
+namespace KafkaJanitor.IntegrationTests
+{
+    public class TopicCreatedScenario
+    {
+        private readonly ITopicRepository _topicRepository;
+
+        [Fact]
+        public async Task TopicCreatedScenarioRecipe()
+        {
+            await When_a_topic_creation_is_requested();
+            await Then_a_topic_is_created();
+            var topic = await Then_the_topic_is_retrieved();
```

Comment:
I feel we need to go to great lengths to ensure the recipe is easy to read. 
I suggest we set propperties for the class in the methods called instead of setting variables in the recipe
[Here is a example](https://github.com/dfds/capability-service/blob/470a9a5295e00e607d6cb90801def1e7cae70882/src/CapabilityService.IntegrationTests/CapabilityService.IntegrationTests/Features/Topics/CreateTopicScenario.cs)